CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
-## 0.7.0 - 2018-05-02
+## 0.7.0 - 2018-05-04
```

Comment:
You know you still have to update the version in package.json, right?

Reply:
@PrashanthCorp You forget something in your PR? :-)

Reply:
I'll take care of it .

Reply:
{
	"name": "vscode-cosmosdb",
	"version": "0.7.0",